server/manager/src/main/java/org/apache/accumulo/manager/tableOps/compact/CompactionDriver.java

```diff
@@ -257,8 +257,13 @@ public int updateAndCheckTablets(Manager manager, FateId fateId)
             otherSelected++;
           }
         } else {
-          // ELASTICITY_TODO if there are compactions preventing selection of files, then add
+          // If there are compactions preventing selection of files, then add
           // selecting marker that prevents new compactions from starting
+          var mutator = tabletsMutator.mutateTablet(tablet.getExtent()).requireAbsentOperation()
+              .requireSame(tablet, ECOMP).putUserCompactionRequested(fateId);
+          mutator.submit(tm -> tm.getUserCompactionsRequested().contains(fateId));
+
+          // Add marker here
```

Comment:
This comment is not related to this code, just this class. At the end of this class there is code to cleanup on failure that could cleanup this new column.

Reply:
I didn't even notice the undo() code, i'll make sure to add that

Reply:
Updated in faf79184b74ac7bbc747f7c8292dc25e7ec01828